mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -384,6 +386,8 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 		ARG 1 invulnerable
 	METHOD m_jplnglbd distanceTo (Lnet/minecraft/unmapped/C_astfners;)F
 		ARG 1 entity
+	METHOD m_jrpsebzm playingPose (Lnet/minecraft/unmapped/C_ufdjspmk;)Z
```

Comment:
What does this do? Is it a setter? If so, `setPlayingPose` would be the proper name

Reply:
Not a setter it's a conditional. According to conventions I should use `isSomething` but `something`

Reply:
nvm. see other comment